docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -92,6 +92,7 @@ Alpha features might be changed or removed without prior notice.
 | `logsContextDatasourceUi`          | Allow datasource to provide custom UI for context view                                                                                                                       |
 | `lokiQuerySplitting`               | Split large interval queries into subqueries with smaller time intervals                                                                                                     |
 | `individualCookiePreferences`      | Support overriding cookie preferences per user                                                                                                                               |
+| `prometheusMetricEncyclopedia`     | Replace the Prometheus builder editor metric select with a paginated and filterable component                                                                                |
```

Comment:
```suggestion
| `prometheusMetricEncyclopedia`     | Replaces the Prometheus query builder metric select option with a paginated and filterable component                                                                                |
```

Reply:
@bohandley , please review my suggestion to ensure I haven't changed the technical meaning of the content. Thank you!